bin/metrics.js

```diff
@@ -0,0 +1,182 @@
+      database: config.General.db_name
+    },
+    patchKey: 'schema-patch-level'
+  })
+  .then(function (db) {
+    return P.all([
+      self.countAccounts(db, lastMidnight),
+      self.countVerifiedAccounts(db, lastMidnight),
+      self.countAccountsWithTwoOrMoreDevices(db, lastMidnight),
+      self.countAccountsWithThreeOrMoreDevices(db, lastMidnight),
+      self.countAccountsWithMobileDevice(db, lastMidnight)
+    ])
+  })
+  .then(function (counts) {
+    fs.appendFileSync('/media/ephemeral0/fxa-admin/basic_metrics.log', JSON.stringify({
+      hostname: os.hostname(),
+      pid: process.pid,
+      op: 'account_totals',
+      total_accounts: counts[0].count,
+      total_verified_accounts: counts[1].count,
+      total_accounts_with_two_or_more_devices: counts[2].count,
+      total_accounts_with_three_or_more_devices: counts[3].count,
+      total_accounts_with_mobile_device: counts[4].count,
+      time: (new Date(lastMidnight)).toISOString(),
+      v: 0
+    }))
+  })
+  .catch(function (error) {
+    log.error({ op: 'metrics.run', err: error })
+  })
+}
+
+function countAccounts (db, createdBefore) {
+  return metricsQuery(db, [
+    'SELECT COUNT(*) AS count',
+    'FROM accounts',
+    'WHERE createdAt < ?;'
+  ].join('\n'), createdBefore)
+}
+
+function countVerifiedAccounts (db, createdBefore) {
+  return metricsQuery(db, [
+    'SELECT COUNT(*) AS count',
+    'FROM accounts',
+    'WHERE createdAt < ?',
+    'AND emailVerified = true;'
+  ].join('\n'), createdBefore)
+}
+
+function countAccountsWithTwoOrMoreDevices (db, createdBefore) {
+  return metricsQuery(db, [
+    'SELECT COUNT(*) AS count',
+    'FROM (',
+    '    SELECT a.uid',
+    '    FROM accounts AS a',
+    '    INNER JOIN sessionTokens AS s',
+    '    ON a.uid = s.uid',
+    '    WHERE a.createdAt < ?',
+    '    GROUP BY (a.uid)',
+    '    HAVING COUNT(s.tokenId) > 1',
+    ') AS sub;'
+  ].join('\n'), createdBefore)
+}
+
+function countAccountsWithThreeOrMoreDevices (db, createdBefore) {
+  return metricsQuery(db, [
+    'SELECT COUNT(*) AS count',
+    'FROM (',
+    '    SELECT a.uid',
+    '    FROM accounts AS a',
+    '    INNER JOIN sessionTokens AS s',
+    '    ON a.uid = s.uid',
+    '    WHERE a.createdAt < ?',
+    '    GROUP BY (a.uid)',
+    '    HAVING COUNT(s.tokenId) > 2',
+    ') AS sub;'
+  ].join('\n'), createdBefore)
+}
+
+function countAccountsWithMobileDevice (db, createdBefore) {
+  return metricsQuery(db, [
+    'SELECT COUNT(*) AS count',
+    'FROM accounts AS a',
+    'INNER JOIN sessionTokens AS s',
+    'ON a.uid = s.uid',
+    'WHERE a.createdAt < ?',
+    'AND s.uaDeviceType = \'mobile\';'
+  ].join('\n'), createdBefore)
+}
+
+function metricsQuery (db, query, createdBefore) {
+  return db.readOneFromFirstResult(query, [ createdBefore ], true)
+}
+
+function trim (string) {
+  return string.trim()
+}
+
+function filterConfig (line) {
+  return line && line[0] !== '#' && line[0] !== ';'
+}
+
```

Comment:
I found plenty of packages on npm that parse a similar format, but we're using `:` instead of `=` as the assignment operator, which threw me at first. If anyone knows of one/some I'm missing, shout.

Or if you guys think it is better to change the config file format so that we can re-use an existing package (or switch to JSON and use `require`) I'm cool with that. I just didn't want to be responsible for that decision in case it was unconventional in the context of the `puppet-config` repo.

In the meantime, I'll add comments here.